ee/allowedcmd/cmd.go

```diff
@@ -31,15 +34,15 @@ func validatedCommand(ctx context.Context, knownPath string, arg ...string) (*ex
 	// We expect to know the exact location for allowlisted commands on all
 	// OSes except for a few Linux distros.
 	if !allowSearchPath() {
-		return nil, fmt.Errorf("not found: %s", knownPath)
+		return nil, fmt.Errorf("%w: %s",ErrCommandNotFound, knownPath)
 	}
 
 	cmdName := filepath.Base(knownPath)
 	if foundPath, err := exec.LookPath(cmdName); err == nil {
 		return newCmd(ctx, foundPath, arg...), nil
 	}
 
-	return nil, fmt.Errorf("%s not found at %s and could not be located elsewhere", cmdName, knownPath)
+	return nil, fmt.Errorf("%w: %s and could not be located elsewhere", ErrCommandNotFound, knownPath)
```

Comment:
Another nitpick -- I think the error message will read more clearly like this:

```suggestion
	return nil, fmt.Errorf("%w: not found at %s and could not be located elsewhere", ErrCommandNotFound, knownPath)
```